spark-plugins/docs/LogisticRegressionClassifier-sparkcompute.md

```diff
@@ -0,0 +1,70 @@
+# LogisticRegressionClassifier
+
+
+Description
+-----------
+Loads a Logistic Regression model from a file of a FileSet dataset and uses it to classify records.
```

Comment:
it has to be file-based.
Maybe PartitionedFileSet would also work, but I think its fine to support just FileSet for now.